other/react-native/jest.config.js

```diff
@@ -0,0 +1,9 @@
+const jestConfig = require('kcd-scripts/config').jest
+
+module.exports = Object.assign(jestConfig, {
+  preset: 'react-native',
+  testEnvironment: 'node',
```

Comment:
This song and dance felt pretty gnarly, but this is the base config I've found that allows for sharing of `kcd-scripts/config` and supporting React Native in `react-test-renderer`. This ensures that the `react-native` module gets transformed by Babel, tells Jest to look for sources and tests from the root, but then scopes tests down to only `other/react-native/__tests__`.